### PR TITLE
Update jira extension

### PR DESCRIPTION
**Pull request by Silv-1**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/jira/src/components/IssueActions.tsx [hunk 1]
```diff
@@ -169,6 +169,15 @@ export default function IssueActions({
       </ActionPanel.Section>
 
       <ActionPanel.Section>
+        {issue.fields.parent && (
+          <Action.Push
+            target={<IssueDetail initialIssue={issue.fields.parent} issueKey={issue.fields.parent?.key} />}
+            title="Open Parent Issue"
+            icon={Icon.ChevronUp}
+            shortcut={{ modifiers: ["cmd", "shift"], key: "n" }}
```
Comment (thomaslombart) [inline extensions/jira/src/components/IssueActions.tsx:177]: This shortcut is already used to add a new comment. Could you use another one? 🙂

Comment (Silv-1) [reply extensions/jira/src/components/IssueActions.tsx:177]: Of course. Which one do you think would be suitable?

Comment (Silv-1) [reply extensions/jira/src/components/IssueActions.tsx:177]: What about using the same shortcut as for `Open Child Issues`? I assume the actions should not be displayed at the same time as each issue either is a subtask and has a parent or is an issue and has subtasks.

Comment (Silv-1) [reply extensions/jira/src/components/IssueActions.tsx]: @thomaslombart The shortcut of the new command `Open Parent Issue` is now set to `cmd` + `shift` + `o` . It's the same shortcut as for the action `Open Child Issues`.

Comment (Silv-1) [reply extensions/jira/src/components/IssueActions.tsx]: It seems issues can have children and a parent at the same time. I changed the shortcut to `cmd` + `shift` + `u`.

**Pull request merged**